scripts/build_ngx_pagespeed.sh

```diff
@@ -425,26 +435,28 @@ Not deleting $directory; name is suspiciously short.  Something is wrong."
   run tar -xzf $(basename "$psol_url")  # extracts to psol/
 
   if "$DYNAMIC_MODULE"; then
-    configure_args="--add-dynamic-module=$nps_module_dir $PS_NGX_EXTRA_FLAGS"
+    add_module="--add-dynamic-module=$nps_module_dir"
   else
-    configure_args="--add-module=$nps_module_dir $PS_NGX_EXTRA_FLAGS"
+    add_module="--add-module=$nps_module_dir"
   fi
+  configure_args=("$add_module" "${extra_flags[@]}")
 
   echo
   if [ -z "$NGINX_VERSION" ]; then
     # They didn't specify an nginx version, so we're just preparing the
     # module for them to install.
     echo "ngx_pagespeed is ready to be built against nginx."
     echo "When running ./configure pass in:"
-    echo "  $configure_args"
-    if [ -z "$PS_NGX_EXTRA_FLAGS" ]; then
+    echo "  $(quote_arguments "${configure_args[@]}")"
```

Comment:
We need to quote them, because the person is going to paste them on the command line, and we don't want it to retokenize when they do that.

Now, with the current set of configure_args there aren't any spaces within arguments, so it's moot, but quoting is more future proof.